pysr/test/test.py

```diff
@@ -513,6 +513,282 @@ def test_jl_function_error(self):
+        X = [1, 1]
+        for i in range(2, 100):
+            X.append(X[i - 1] + X[i - 2])
+        X = np.asarray(X).reshape(-1, 1)
+        model = PySRSequenceRegressor(recursive_history_length=3)
+        with self.assertRaises(ValueError) as cm:
+            model.fit(X, complexity_of_variables=[1])
+
+        self.assertIn(
+            "number of elements in `complexity_of_variables`", str(cm.exception)
+        )
+
+    def test_sequence_error_message_both_variable_complexity(self):
+        X = [1, 1]
+        for i in range(2, 100):
+            X.append(X[i - 1] + X[i - 2])
+        X = np.asarray(X).reshape(-1, 1)
+        model = PySRSequenceRegressor(
+            **self.default_test_kwargs, complexity_of_variables=[1, 2]
+        )
+        with self.assertRaises(ValueError) as cm:
+            model.fit(X, complexity_of_variables=[1, 2, 3])
+
+        self.assertIn(
+            "You cannot set `complexity_of_variables` at both `fit` and `__init__`.",
+            str(cm.exception),
+        )
+
+    def test_sequence_warm_start_set_at_init(self):
+        # Smoke test for bug where warm_start=True is set at init
+        X = [1, 1, 1]
+        for i in range(3, 30):
+            X.append(X[i - 1] + X[i - 2] + X[i - 3])
+        X = np.asarray(X).reshape(-1, 1)
+        regressor = PySRSequenceRegressor(
+            **self.default_test_kwargs, warm_start=True, max_evals=10
+        )
+        regressor.fit(X)
+
+    def test_sequence_noisy_builtin_variable_names(self):
+        X = [1, 1]
+        for i in range(2, 30):
+            X.append(X[i - 1] + X[i - 2])
+        X = np.asarray(X).reshape(-1, 1)
+        model = PySRSequenceRegressor(
+            binary_operators=["+"],
+            **self.default_test_kwargs,
+            early_stop_condition="stop_if(loss, complexity) = loss < 0.05 && complexity == 2",
+        )
+        # We test builtin variable names
+        model.fit(X, variable_names=["exec"])
+        self.assertLessEqual(model.get_best()["loss"], 1e-2)
+        self.assertLessEqual(model.get_best()["loss"], 1e-2)
+        self.assertIn("exec", model.latex())
+
+    def test_sequence_multidimensional_data_error(self):
+        X = [
+            [
+                [1, 2],
+                [3, 4],
+                [5, 6],
+            ],
+            [
+                [7, 8],
+                [9, 10],
+                [11, 12],
+            ],
+            [
+                [13, 14],
+                [15, 16],
+                [17, 18],
+            ],
+        ]
+        for i in range(3, 5):
+            X.append(
+                [
+                    [X[i - 1][0][0] + X[i - 2][0][1], X[i - 1][0][1] + X[i - 2][1][0]],
+                    [X[i - 1][1][0] + X[i - 3][1][1], X[i - 1][1][1] + X[i - 3][1][0]],
+                    [X[i - 2][2][0] + X[i - 3][2][1], X[i - 2][2][1] + X[i - 3][2][0]],
+                ]
+            )
+        X = np.asarray(X)
```

Comment:
```suggestion
        X = np.zeros((10, 1, 1))
```

Since it's just testing the presence of an error, can just do something like this